openapi.yaml

```diff
@@ -6918,6 +6974,125 @@ paths:
+      description: >
+        Returns the values for a specified node object.
+      responses:
+        '200':
+          description: Returns the values of a node object in the form that it appears currently in the node pool array.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  data:
+                    type: object
+                    description: >
+                      The selected node in the cluster.
+                    properties:
+                      id:
+                        type: string
+                        readOnly: true
+                        description: >
+                          The Node's ID.
+                        example: "12345-6aa78910bc"
+                      instance_id:
+                        type: integer
+                        description: >
+                          The Linode's ID. If no Linode is currently provisioned for this Node, this is `null`.
+                        example: 123456
+                      status:
+                        type: string
+                        description: >
+                          The Node's status as it pertains to being a Kubernetes node.
+                        enum:
+                        - ready
+                        - not_ready
+                        example: ready
+        default:
+          $ref: '#/components/responses/ErrorResponse'
+      x-code-samples:
+      - lang: Shell
+        source: >
+          curl -H "Authorization: Bearer $TOKEN" \
+                https://api.linode.com/v4/lke/clusters/12345/nodes/12345-6aa78910bc
+      - lang: CLI
+        source: >
+          linode-cli lke node-view 123456 12345-6aa78910bc
+  /lke/clusters/{clusterId}/nodes/{nodeId}/recycle:
+    parameters:
+    - name: clusterId
+      in: path
+      description: ID of the Kubernetes cluster containing the Node.
+      required: true
+      schema:
+        type: integer
+    - name: nodeId
+      in: path
+      description: ID of the Node to be recycled.
+      required: true
+      schema:
+        type: string
+    x-linode-cli-command: lke
+    post:
+      operationId: postLKEClusterNodeRecycle
+      x-linode-cli-action: node-recycle
+      security:
+      - personalAccessToken: []
+      - oauth:
+        - lke:read_write
+      tags:
+      - Linode Kubernetes Engine (LKE)
+      summary: Node Recycle
+      description: |
+        Recycles an individual Node in the designated Kubernetes Cluster. The Node will be deleted
+        and replaced with a new Linode, which may take a few minutes. Replacement Nodes are
+        installed with the latest available patch for the Cluster's Kubernetes Version.
+
+        **Any local storage on deleted Linodes (such as "hostPath" and "emptyDir" volumes, or "local" PersistentVolumes) will be erased.**
+      responses:
+        '200':
+          description: Node has been recycled.
```

Comment:
When you get a 200 OK from this endpoint, it doesn't mean the node has been recycled, but rather that the request to recycle the node has succeeded and is now in progress.